x-pack/plugins/security_solution/public/timelines/components/timeline/properties/helpers.tsx

```diff
@@ -36,12 +30,28 @@ interface SmallNotesButtonProps {
   toggleShowNotes: (eventId?: string) => void;
   timelineType: TimelineTypeLiteral;
   eventId?: string;
+  notesCount: number;
 }
 
 export const NOTES_BUTTON_CLASS_NAME = 'notes-button';
 
+export const NotificationDot = styled.span`
+  position: absolute;
+  display: block;
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background-color: ${({ theme }) => theme.eui.euiColorDanger};
+  top: 15%;
+  left: 65%;
```

Comment:
you'll see that [I tweaked](https://github.com/elastic/kibana/pull/186931/files?show-viewed-files=true&file-filters%5B%5D=#diff-21ee437e57c1f2707c69310d046589b9eb26421b83477bbe400a179f030c40d9R30-R31) those values ever so slightly. This was just personal preference on how it looked. If you don't like my values, I have no issues using yours!